examples/src/pages/Migration.tsx

```diff
@@ -89,6 +90,13 @@ export function Migration() {
       viewer = new Cognite3DViewer(viewerOptions);
       (window as any).viewer = viewer;
 
+      const controlsOptions: CameraControlsOptions = {
+        onClickTargetChange: true,
+        zoomToCursor: 'scrollTarget'
+      }
+
+      viewer.setCameraControlsMode(controlsOptions);
```

Comment:
This looks really nice. Could you also add some stuff to the Migration UI to allow setting different options so it's easy to test out the different modes there?

Reply:
Added